### ISSUE TITLE
Can't be build from the sources using Java 10

### ISSUE THREAD

**Issue by vict0r81** [aws/aws-toolkit-jetbrains#425]
**Describe the bug**
Can't be built from the sources, using Java 10
`java -version`
> java version "10.0.2" 2018-07-17
> Java(TM) SE Runtime Environment 18.3 (build 10.0.2+13)
> Java HotSpot(TM) 64-Bit Server VM 18.3 (build 10.0.2+13, mixed mode)

`uname -a`
> Darwin vmbpro.local 18.2.0 Darwin Kernel Version 18.2.0: Fri Oct  5 19:41:49 PDT 2018; root:xnu-4903.221.2~2/RELEASE_X86_64 x86_64

**To reproduce**
1. Download source code
2. Build using steps indicated in README document using gradle
3. Scroll down to the build results.
4. See error:
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> A problem occurred configuring project ':jetbrains-core'.
> > Failed to notify project evaluation listener.
>    > javax/xml/bind/JAXBContext
> * Try:
> Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> BUILD FAILED in 4m 20s

**Expected behavior**
No issue during the build process.

**Screenshot**
![bug_report_screen](https://user-images.githubusercontent.com/8981423/48344494-e578cd80-e67d-11e8-829b-b151c5855293.png)

**Your Environment**
 - OS: macOS 10.14.1
 - JetBrains' Product: Intellij IDEA
 - JetBrains' Product Version:  2018.2
 - Toolkit Version: 0.1-SNAPSHOT'

**Additional context**

**Comment by abrooksv** [comment]
Hey, thanks for trying to build the plugin. It currently requires Java 8 to build.

**Comment by vict0r81** [comment]
@abrooksv Hey, m.b. we can convert to Feature request? 'cause as you know, currently Java 11 is already released. So the support of new version would be a good thing. Don't you think?

But, for now, as a part of workaround, here's the solution I've used for successful build, using Java 10 SKD.
instead of plain
`./gradlew buildPlugin`

I've used:
`GRADLE_OPTS="--add-modules java.xml.bind '-Dorg.gradle.jvmargs=--add-modules java.xml.bind'" ./gradlew buildPlugin`

So basically it's a standard build issue during switch from Java 8 to Java 9 and above.

Or, add those parameters directly to `gradle.build` file, see Gist line 115
[gradle.build for JDK 9 and above](https://gist.github.com/vict0r81/02383ebc61ab9172ec16ac864b97228f)
